src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div>
+      <form
+        action="http://localhost:8080/"
+        method="post"
+        class="firstSearchString"
+        data-qa="big"
+        >
```

Comment:
```suggestion
      <form
        action="http://localhost:8080/"
        method="post"
        class="firstSearchString"
        data-qa="big"
      >
```

Reply:
read checklist and fix code style everywhere

Reply:
done